testing/fixtures/execute_cmd_fixtures.go

```diff
@@ -51,8 +59,7 @@ commands:
   - build: echo building project
 `
 
-	expectedOutput := `No command 'random' found in config file
-
+	expectedOutput := aurora.Red("\nError building exectuion plan. Command \"random\" not found.").Bold().String() + `
```

Comment:
Do you think checking color in test is worth and necessary - I am ok to just test the output and not colors.

WDYT?